app/jobs/webhooks/facebook_events_job.rb

```diff
@@ -3,7 +3,7 @@ class LockAcquisitionError < StandardError; end
 
   queue_as :default
   # https://edgeapi.rubyonrails.org/classes/ActiveJob/Exceptions/ClassMethods.html#method-i-retry_on
-  retry_on LockAcquisitionError, wait: 2.seconds, attempts: 5
+  retry_on LockAcquisitionError, wait: proc { |attempt| 2**attempt }, attempts: 5
```

Comment:
@sojan-official one problem with an exponential backoff is that the sequence in which the events are processed might get affected. 

Should I change it to a fixed time, say `4 seconds` instead of the original `2 seconds`?

```suggestion
  retry_on LockAcquisitionError, wait: 4.seconds, attempts: 5
```